### PR TITLE
Adds a v-field-select component

### LINKED ISSUE #225: Recommend fields on all relational interfaces/displays.
Having predefined fields to select would be super helpful when setting up the interfaces / displays.
At the moment you don't really always know what fields there exist in a collection.
grafik

### PR DESCRIPTION
**Pull request by Nitwel**
Here is a short preview of how it looks currently.
GIF 11 09 2020 14-11-46

Closes #225

### CODE REVIEW

**Comment by Nitwel** [comment]
Not to me. Remove that component from the layout when done!

**Comment by benhaynes** [comment]
Shifting to a tags styling...

Screen Shot 2020-09-11 at 9 06 41 AM

File: app/src/components/v-field-select/v-field-select.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		<v-chip
+			v-for="(field, index) in activeFields"
+			:key="index"
+			class="field draggable"
+			@click.stop="removeField(field.field)"
+		>
+			{{ field.name }}
+		</v-chip>
+		<v-menu
+			showArrow
```
Comment (rijkvanzanten) [inline app/src/components/v-field-select/v-field-select.vue:12]:
```suggestion
			show-arrow
```

File: app/src/interfaces/many-to-many/options.vue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+				});
+			},
+		});
+
+		const collection = computed(() => {
+			if (props.fieldData.field == null || props.fieldData.meta?.collection == null) return null;
+			const junction = relationsStore.getRelationsForField(
+				props.fieldData.meta.collection,
+				props.fieldData.field
+			)?.[0];
```
Comment (rijkvanzanten) [inline app/src/interfaces/many-to-many/options.vue:47]: It's not a given that the first of the two is always the junction one. We'll have to double check the actual relation to make sure it's the o2m, and not the m2o

File: app/src/interfaces/one-to-many/options.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+				});
+			},
+		});
+
+		const collection = computed(() => {
+			if (props.fieldData.field == null || props.fieldData.meta?.collection == null) return null;
+			const relationData = relationsStore.getRelationsForField(
+				props.fieldData.meta.collection,
+				props.fieldData.field
+			)?.[0];
```
Comment (rijkvanzanten) [inline app/src/interfaces/one-to-many/options.vue:49]: Same here

File: app/src/interfaces/one-to-many/options.vue [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+			return relationData.many_collection;
```
Comment (rijkvanzanten) [inline app/src/interfaces/one-to-many/options.vue:50]: With the `?.[0]` above, relationData can be undefined

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/interfaces/many-to-many/options.vue [hunk 5]
```diff
@@ -0,0 +1,66 @@
+<template>
+	<v-notice type="warning" v-if="collection == null">
```
Comment (rijkvanzanten) [inline app/src/interfaces/many-to-many/options.vue:2]: Another one @Nitwel :P 

```suggestion
	<v-notice type="warning" v-if="collection === null">
```

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**